src/components/sidebar/sidebar.tsx

```diff
@@ -2,9 +2,13 @@ import React from 'react';
 
 import './sidebar.scss';
 
-class Sidebar extends React.Component {
-    constructor() {
-        super();
+interface SidebarProps {
+    children: React.ReactNode[];
```

Comment:
no need in `ReactNode[]`,
```
    type ReactText = string | number;
    type ReactChild = ReactElement | ReactText;

    interface ReactNodeArray extends Array<ReactNode> {}
    type ReactFragment = {} | ReactNodeArray;
    type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```
see `ReactFragment` and `ReactNodeArray`

Reply:
I agree, I see your point